src/execute/oas3/parameter-builders.js

```diff
@@ -50,7 +50,7 @@ export function query({ req, value, parameter }) {
     value = '0';
   }
 
-  if (value) {
+  if (value && (!Array.isArray(value) || value.length > 0)) {
```

Comment:
Added the same check for `query` despite it working without it - when we had an empty array, it would go inside the if block but we somehow wouldn't have the empty field. Not sure if this change is needed.